packages/manifold/contracts/edition/ManifoldERC721Edition.sol

```diff
@@ -29,101 +29,108 @@ contract ManifoldERC721Edition is CreatorExtension, ICreatorExtensionTokenURI, I
+        return string(abi.encodePacked(_tokenPrefix[creatorCore][instanceId], (index+1).toString()));
     }
     
     /**
      * @dev See {IManifoldERC721Edition-mint}.
      */
-    function mint(address creator, uint256 series, address recipient, uint16 count) external override nonReentrant creatorAdminRequired(creator) {
-        require(count > 0, "Invalid amount requested");
-        require(_totalSupply[creator][series]+count <= _maxSupply[creator][series], "Too many requested");
+    function mint(address creatorCore, uint256 instanceId, address recipient, uint16 count) external override nonReentrant creatorAdminRequired(creatorCore) {
+        if (count == 0) revert("Invalid amount requested");
+        if (_totalSupply[creatorCore][instanceId]+count > _maxSupply[creatorCore][instanceId]) revert ("Too many requested");
         
-        uint256[] memory tokenIds = IERC721CreatorCore(creator).mintExtensionBatch(recipient, count);
-        _updateIndexRanges(creator, series, tokenIds[0], count);
+        uint256[] memory tokenIds = IERC721CreatorCore(creatorCore).mintExtensionBatch(recipient, count);
+        _updateIndexRanges(creatorCore, instanceId, tokenIds[0], count);
     }
 
     /**
      * @dev See {IManifoldERC721Edition-mint}.
      */
-    function mint(address creator, uint256 series, address[] calldata recipients) external override nonReentrant creatorAdminRequired(creator) {
-        require(recipients.length > 0, "Invalid amount requested");
-        require(_totalSupply[creator][series]+recipients.length <= _maxSupply[creator][series], "Too many requested");
+    function mint(address creatorCore, uint256 instanceId, address[] calldata recipients) external override nonReentrant creatorAdminRequired(creatorCore) {
+        if (recipients.length == 0) revert("Invalid amount requested");
+        if (_totalSupply[creatorCore][instanceId]+recipients.length > _maxSupply[creatorCore][instanceId]) revert("Too many requested");
         
-        uint256 startIndex = IERC721CreatorCore(creator).mintExtension(recipients[0]);
+        _mintTokens(creatorCore, recipients, instanceId);
+    }
+
+    function _mintTokens(address creatorCore, address[] memory recipients, uint256 instanceId) private {
+        uint256 startIndex = IERC721CreatorCore(creatorCore).mintExtension(recipients[0]);
         for (uint256 i = 1; i < recipients.length;) {
-            IERC721CreatorCore(creator).mintExtension(recipients[i]);
-            unchecked{i++;}
+            IERC721CreatorCore(creatorCore).mintExtension(recipients[i]);
+            unchecked{++i;}
+        }
+        _updateIndexRanges(creatorCore, instanceId, startIndex, recipients.length);
+    }
+
+    /**
+     * @dev See {IManifoldERC721Edition-createSeries}.
+     */
+    function createSeries(address creatorCore, uint256 maxSupply_, string calldata prefix, uint256 instanceId, address[] memory recipients) external override creatorAdminRequired(creatorCore) returns(uint256) {
```

Comment:
Does this work if I want to initialize a series and mint a bunch of tokens to my address only?